### ISSUE TITLE
u128 function parameters don't show values in debugger

### ISSUE THREAD

**Issue by Kage-Yami** [intellij-rust/intellij-rust#6755]
## Environment

* **IntelliJ Rust plugin version:** 0.3.139.3615-203
* **Rust toolchain version:** 1.49.0 (e1884a8e3 2020-12-29) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2020.3.2 (CL-203.7148.70)
* **Operating system:** Windows 10 10.0
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description

`unsigned __int128` (`u128`) function parameters don't seem to display their value in the debugger's Variables view. After changing the parameter type to something else (e.g. `u64`), the parameter will then display its value in the debugger.

## Steps to reproduce

```rust
fn recurse(n: u128) {
    recurse(n);
}

#[cfg(test)]
mod tests {
    #[test]
    fn recurse_test() {
        crate::recurse(30);
    }
}
```

1. Debug the test in the above code (the recursion will lead to a stack overflow)

In any of the function calls in the stack (except the top one), it will only show `n = {unsigned __int128}` If you change `u128` in the example code to `u64`, then the debugger will show `n = {unsigned long long} 30`.

**Comment by ortem** [comment]
Related to https://github.com/intellij-rust/intellij-rust/issues/6755

**Comment by Kage-Yami** [comment]
@ortem Umm, you've related this ticket to itself...

**Comment by ortem** [comment]
Fixed since CLion 2022.3 with enabled native Rust support in MSVC LLDB (https://github.com/intellij-rust/intellij-rust/pull/9649)
